### PR TITLE
ERR: better error message on unparseable datetimes

### LINKED ISSUE #10720: DOC/ERR: better error message on unsuccessful datetime parsing
exceptions that are raised on unsuccessful datetime/timedelta parsing should add this:

``you can coerce to NaT by passing errors='coerce'``

comment at the end: https://github.com/pydata/pandas/pull/10674

### PR DESCRIPTION
**Pull request by baevpetr**
- [ ] closes #10720
- [ ] tests added / passed
- [x] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by baevpetr** [comment]
So, Linux py36_minimum_versions & macOS py36_macos builds do not pass the test:
their exceptions don't seem to contain the passing date.

**Comment by simonjayhawkins** [comment]
@baevpetr can you continue pushing to #30711 instead of opening a new PR.

**Comment by baevpetr** [comment]
@simonjayhawkins, no, I can't. Sorry.
image

**Comment by simonjayhawkins** [comment]
If you now unable to reinstate the original repository, then you should put a note in the original PR to make the reviewers of the original PR aware. To help maintain the history, should also update the OP to reference the previous PR.

File: pandas/tests/indexes/datetimes/test_tools.py [hunk 1]
```diff
@@ -480,6 +480,17 @@ def test_to_datetime_unparseable_ignore(self):
         s = "Month 1, 1999"
         assert pd.to_datetime(s, errors="ignore") == s
 
+    def test_to_datetime_unparseable_raise(self):
+        # GH#10720
+        date = "Month 1, 1999"
+        expected_args = (
+            f"Unknown string format: %s {date}. "
```
Comment (MomIsBestFriend) [inline pandas/tests/indexes/datetimes/test_tools.py]: I believe that the error is here, can you remove the ```%s```?

Comment (jbrockmendel) [reply pandas/tests/indexes/datetimes/test_tools.py]: this looks likely

File: pandas/tests/indexes/datetimes/test_tools.py [hunk 2]
```diff
@@ -480,6 +480,17 @@ def test_to_datetime_unparseable_ignore(self):
         s = "Month 1, 1999"
         assert pd.to_datetime(s, errors="ignore") == s
 
+    def test_to_datetime_unparseable_raise(self):
+        # GH#10720
+        date = "Month 1, 1999"
+        expected_args = (
```
Comment (jreback) [inline pandas/tests/indexes/datetimes/test_tools.py]: call this msg

File: pandas/tests/indexes/timedeltas/test_tools.py [hunk 3]
```diff
@@ -110,6 +110,16 @@ def test_to_timedelta_invalid(self):
             invalid_data, to_timedelta(invalid_data, errors="ignore")
         )
 
+        # GH#10720
```
Comment (jreback) [inline pandas/tests/indexes/timedeltas/test_tools.py]: can you make a new test

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/indexes/datetimes/test_tools.py (hunk 1 above)
Comment (baevpetr) [reply pandas/tests/indexes/datetimes/test_tools.py]: `AssertionError: Pattern "Unknown string format: %s Month 1, 1999. You can coerce to NaT by passing errors='coerce'" not found in "Unknown string format. You can coerce to NaT by passing errors='coerce'"`
That is output from tests on Linux py36_minimum_versions & macOS py36_macos. There's no date at all. On other platforms test passes.

File: pandas/tests/indexes/datetimes/test_tools.py (hunk 2 above)
Comment (baevpetr) [reply pandas/tests/indexes/datetimes/test_tools.py]: Ok.

File: pandas/tests/indexes/timedeltas/test_tools.py (hunk 3 above)
Comment (baevpetr) [reply pandas/tests/indexes/timedeltas/test_tools.py]: `timedeltas` passes normally. What logic should I test? (for now moved that test to other tests with `'coerce'`).

**Comment by baevpetr** [comment]
And suddenly `pip-20` problems with installation in CI...

**Comment by WillAyd** [comment]
@baevpetr is this still active? Can you try to get CI green?

**Comment by WillAyd** [comment]
Closing as stale but ping if you'd like to pick back up

**Pull request closed without merging**